src/User.php

```diff
@@ -46,8 +97,10 @@ public function getPhotoUrl()
     public function toArray()
     {
         return [
+        'email' => $this->email,
         'name' => $this->name,
-        'photourl' => $this->photoUrl
+        'photourl' => $this->photoUrl,
```

Comment:
@j-feng13 I think we need to default these values to an empty string if they are not set.